### ISSUE TITLE
mount does not work for LUKS partitions

### ISSUE THREAD

**Issue by carlos22** [MatMoul/archfi#26]
If the root is crypted, then the "mount exsisting" should call `cryptsetup luksOpen /dev/<id> root` and then `mount /dev/mapper/root /mnt`. This is useful if you run the `archfi` multiple times (e.g. if you forgot something). Great script :+1:

**Comment by MatMoul** [comment]
You'r right, archfi is made to be launched multiple time and crypted partition is not handled on second mount.
At this time, you can mount your partition manualy with a second TTY.
I don't have tested how I can detect a crypted part, I'll try to find a solution.

**Comment by carlos22** [comment]
I think the parition type is "e8" LUKS (see: e8 https://www.win.tue.nl/~aeb/partitions/partition_types-1.html).

Easy method to find out (as root):
```
$ blkid -o value -s TYPE /dev/sda4
crypto_LUKS
```

**Comment by MatMoul** [comment]
Thanks for the tricks... I'll try to add it to the script

**Comment by MatMoul** [comment]
Sorry for the late response to this request ...
I made a lot of changes on archfi which prepares the possibility to manage the detection of luks ...

**Comment by TheLinuxNinja** [comment]
Is this something you need me to work on? When exiting archfi, any luksOpen commands that were run should be followed up with a luksClose to be able to leave a clean slate for the next archfi run.

**Comment by MatMoul** [comment]
@TheLinuxNinja You're right, on install when you reboot, it isn't a problem but if you use it on an existing system, this can be a problem...
